browser/ts/index.ts

```diff
@@ -0,0 +1,370 @@
+import { SignedOrder } from '@0x/order-utils';
+import { wasmBuffer } from './generated/wasm_buffer';
+import { BigNumber } from '@0x/utils';
+
+export { SignedOrder } from '@0x/order-utils';
+export { BigNumber } from '@0x/utils';
+
+// Side-effect import
+// Side-effects include adding an `fs` and `Go` property on the global object.
+import './wasm_exec';
+
+// The Go code sets certain global values and this is our only way of
+// interacting with it. Define those values and their types here.
+declare global {
+    // Defined in wasm_exec.ts
+    class Go {
+        run(instance: WebAssembly.Instance): void;
+        importObject: any;
+    }
+
+    // Defined in ../go/main.go
+    const zeroExMesh: ZeroExMesh;
+}
+
+/**
```

Comment:
@fabioberger We should probably generate docs for the TypeScript bindings similar to what we did for the TypeScript RPC client. Can you point me in the right direction? Alternatively we could do that in a separate PR since this is already pretty big.

Reply:
You'll need `"@0x/ts-doc-gen": "^0.0.16",` dep and a command:

```
"scripts": {
    "docs:md": "ts-doc-gen --sourceDir=./src --output=${npm_package_config_docsPath}"
},
"config": {
    "docsPath": "../../../docs/json_rpc_clients/typescript"
},
```

Just change the `docsPath` to where you want the MD generated to and it should work 👍

Reply:
👍@fabioberger just added this. Mind eyeballing the docs to make sure everything looks right?